Warehouse.Common/Extensions/StringExtension.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Warehouse.Common.Extensions
+{
```

Comment:
Since we are using the latest framework let's use this approach for all files. This comment applies to all files:
[learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces)